resources/views/admin/partials/forums/forum_form.twig

```diff
@@ -0,0 +1,114 @@
+            <input type="text" name="slug" id="slug" value="{{ forum ? forum.slug : old('slug') }}">
+        </div>
+        <div class="form__row">
+            <h3 class="form__row__title">
+                <label for="description"> {{ trans('admin::forums.description') }} </label>
+            </h3>
+            <textarea name="description" id="description">{{ forum ? forum.description : old('description') }}</textarea>
+        </div>
+        {% if forums.count > 0 %}
+            <div class="form__row">
+                {# TODO js hide if type == category #}
+                <h3 class="form__row__title">
+                    <label for="parent"> {{ trans('admin::forums.parent') }} </label>
+                </h3>
+                <p class="form__row__description">{{ trans('admin::forums.parent.desc') }}</p>
+                <select id="parent" name="parent">
+                    {% if forum is defined and forum.parent_id %}
+                        {{ list_all_forums(forums, 'admin.forums.forum-item-select', {'active': forum.parent_id}) }}
+                    {% else %}
+                        {{ list_all_forums(forums, 'admin.forums.forum-item-select') }}
+                    {% endif %}
+                </select>
+            </div>
+            <div class="form__row">
+                {# TODO js show only children of parent forum #}
+                {# TODO js forums order #}
+                {# Maybe something different than select? IMHO sortable items would be nice concept http://i.imgur.com/KyLvVIL.png #}
+                <h3 class="form__row__title">
+                    <label for="order"> {{ trans('admin::forums.order') }} </label>
+                </h3>
+                <p class="form__row__description">{{ trans('admin::forums.order.desc') }}</p>
+                <div class="segmented-control">
+                                <span class="segmented-control__option">
+                                    <input id="order_before" class="segmented-control__option__input js-forum_type" name="order_position" type="radio" value="0" {# warningType.must_acknowledge == 1 or not(warningType is defined) ? 'checked="checked"' #}>
+                                    <label class="segmented-control__option__button" for="order_before">{{ trans('admin::forums.before') }}</label>
+                                </span>
+                    <span class="segmented-control__option">
+                                    <input id="order_after" class="segmented-control__option__input js-forum_type" name="order_position" type="radio" value="1" checked{# warningType is defined and warningType.must_acknowledge == 0 ? 'checked="checked"' #}>
+                                    <label class="segmented-control__option__button" for="order_after">{{ trans('admin::forums.after') }}</label>
+                                </span>
+                </div>
+                <select id="order" name="order">
+                    <option value="0">NuLL</option>
```

Comment:
The whole forum sorting need rewrite using javascript. It's placeholder so I leave this as is for now